grafana/grafana
# Snapshots: Fix usage of sign in link from the snapshot page

**What this PR does / why we need it**:
Fix redirect to login page from snapshot page when not authenticated.

**Which issue(s) this PR fixes**:
Fixes #28547 

**Special notes for your reviewer**:
How to test:
1. Not authenticated and anonymous auth enabled or not 
2. Open a snapshot
3. Click on Sign in link in sidemenu
4. Should be redirected to login page
5. Login should redirect you to the originating snapshot

Do you see any potential security implications with my suggested fix?
